desktop/linux/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM ghcr.io/cirruslabs/flutter:dev
+FROM cirrusci/flutter
```

Comment:
I know nothing about docker but shouldn't this be the new image `ghcr.io/cirruslabs/flutter`?